blender/LilySurfaceScrapper/Scrappers/CgbookcaseScrapper.py

```diff
@@ -73,9 +88,15 @@ def fetchVariant(self, variant_index, material_data):
             'Roughness': 'roughness',
             'Metallic': 'metallic',
         }
-        for m in v.xpath(".//a[@class='directdownload']"):
-            map_name = m.xpath("div[@class='downloadable']/text()")[0].strip()
+        for m in v.xpath(".//a"):
             map_url = "https://www.cgbookcase.com" + m.attrib['href']
+
+            temp = map_url[map_url.find("K_") + 2:-4].split("_")[-1::-1]
+            map_name = " ".join(map_url[map_url.find("K_") + 2:-4].split("_")[-1::-1]).title()
```

Comment:
This line is overwritten by teh next one anyway.

```suggestion
            map_name = " ".join(temp).title()
```

Reply:
oh lol